### ISSUE TITLE
Remove usages of uniqueGroupId

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/inverse-square-law-common#61]
From https://github.com/phetsims/scenery-phet/issues/485, this one was just a little trickier because gravity-force-lab is using one of the same "uniqueGroupId"s from inverse-square-law-common. I think the change was straight forward but @zepumph can you take a a look since you have been working on this sim recently?

**Comment by jessegreenberg** [comment]
Sorry, I broke something, not quite ready.

**Comment by jessegreenberg** [comment]
Alright, should be OK.

**Comment by jessegreenberg** [comment]
In order to reference the utterances as instance variables I made some of the functions no longer static, hope that is OK. It seemed reasonable because I noticed some other alert functions were already not static.

**Comment by zepumph** [comment]
Those commits look totally fine to me. Thanks for taking care of this.
